recipes/bumpver/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "bumpver" %}
+{% set version = "2021.1112" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/bumpver-{{ version }}.tar.gz
+  sha256: 715dd9e8526609de2008a9c62ec80d891af41055f95e6c328a8ca6dc8f0fa54a
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - click
+    - colorama >=0.4
+    - lexid
+    - pathlib2
+    - setuptools >=45.0.0
+    - toml
+    - typing
```

I think you can drop this `typing` requirement if you go to `noarch: python` and require `python >=3.5`.

---

the package is >=2.7. I added the version numbers

---

@dopplershift has a point. The package uses an environment marker `python_version < "3.5"` so that the `typing` package is not installed on newer versions of python (where it's already part of the standard library).

I imagine the same would work here:

```suggestion
    - typing; python_version < "3.5"
```

---

Unfortunately, having what we call a "selector" on typing would mean that this package can't be `noarch: python`, and the machinery would have to build packages for each of the architectures/python versions. That seems excessive for this straightforward, pure python package.

@raybellwaves as the recipe maintainer (and I assume user), how important is the support for Python 2.7? We're not building compiled packages for 2.7 by default any more. If you don't need it, I would just make it depend on >=3.5 and drop `typing`.

I'm also happy to merge as is (just unconditionally installing `typing`) too.

---

i'll be using it with 3.8.